garak/resources/gcg/attack_manager.py

```diff
@@ -1429,7 +1431,7 @@ def run(
         test_steps: int = 50,
         incr_control: bool = True,
         stop_on_success: bool = True,
-        verbose: bool = True,
+        verbose: bool = False,
```

Comment:
can this be taken from `garak._config.system.verbose`?

Reply:
It can! That's a good one.